### PR TITLE
Fix PRNG generation

### PR DESCRIPTION
**Pull request by SonicMastr**
This fixes errors for requests that require PRNG generation. (ie. Youtube-dl requests)

### CODE REVIEW

File: crypto/rand/rand_unix.c [hunk 1]
```diff
@@ -241,6 +239,18 @@ int RAND_poll(void)
 
     return 1;
 }
+# elif defined __vita__
```
Comment (d3m3vilurr) [inline crypto/rand/rand_unix.c]:
```suggestion
# elif defined(__vita__)
```

File: crypto/rand/rand_unix.c [hunk 2]
```diff
@@ -241,6 +239,18 @@ int RAND_poll(void)
...
+{
+    unsigned char buf[ENTROPY_NEEDED];
+
+    sceKernelGetRandomNumber((void *)buf, sizeof(buf));
+    RAND_add(buf, sizeof(buf), ENTROPY_NEEDED);
+    OPENSSL_cleanse(buf, sizeof(buf));
+
+    return 1;
+}
 # else                          /* !defined(__OpenBSD__) */
```
Comment (d3m3vilurr) [inline crypto/rand/rand_unix.c]:
```suggestion
# else                          /* !defined(__OpenBSD__) && !defined(__vita__) */
```

**Review by d3m3vilurr — CHANGES_REQUESTED** [review]

**Review by SonicMastr — COMMENTED** [review]
Added the commits

**Comment by d3m3vilurr** [comment]
thanks :+1:

**Pull request merged**